### DISCUSSION TITLE
Display battery level

### DISCUSSION THREAD

**Discussion by Cooper-Han** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1084] [Ideas]
After using AIDente Pro, I removed the system battery icon from the menu bar on my Mac and replaced it with the AIDente Pro menu bar icon. However, I do not want the battery percentage to be displayed in the menu bar as it takes up too much space. Instead, I would like to be able to click on the battery icon in the menu bar to view the current battery percentage.

**Comment by MatthiasKerbl** [comment]
Hi @HKZ773999,

Good idea. This feature will be added in one of the upcoming updates.

Best,
Matthias
